core/src/uix/core.cljs

```diff
@@ -213,3 +213,53 @@
   (let [lazy-component (react/lazy #(.then (f) (fn [component] #js {:default component})))]
     (set! (.-uix-component? lazy-component) true)
     lazy-component))
+
+(defn react-component-ctor []
+  (fn [^js/React.Component this _]
+    (set! (.-state this) #js {:argv nil})
+    (specify! (.-state this)
```

Comment:
I'd say for UIx2 we should stick with hook-like API, use a tuple of `[value set-value]` instead of Atom-like wrapper, to be consistent with `use-state` hook. For UIx1 it made sense since local state there is wrapped as an atom.

Reply:
Makes sense! I'll action